src/visualization/index.ts

```diff
@@ -1,56 +1,19 @@
-import {FunctionComponent, ComponentClass} from 'react'
-import {AnnotationsList, ViewProperties, TimeRange} from 'src/types'
-import {FluxResult} from 'src/types/flows'
-import {SimpleTableViewProperties} from 'src/visualization/types/SimpleTable'
+import {
```

Comment:
why were the definitions moved to the context loader?

Reply:
the context loader depends on them and makes use of them. They're exported here to maintain the Interface that visualizations/index.ts exports